Sources/Common/Store/SdkConfig.swift

```diff
@@ -25,7 +25,6 @@ public struct SdkConfig {
                 siteId: siteId,
                 apiKey: apiKey,
                 region: region,
-                trackingApiUrl: region.productionTrackingUrl,
```

Comment:
Referring to the [comment above](https://github.com/customerio/customerio-ios/blob/d7836fc0fecdd74384854b96a75e538379ac9ad2/Sources/Common/Store/SdkConfig.swift#L15), there's an example using `trackingApiUrl` to modify the `trackingApiUrl`.  I'm wondering if we could consider removing it to avoid confusion for users?